`kraft fetch` panics immediately

### Describe the bug

```console
$ kraft fetch
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5557117f0cce]
```

--- a/internal/cli/kraft/events/events.go
+++ b/internal/cli/kraft/events/events.go
@@ -47,7 +47,8 @@ func NewCmd() *cobra.Command {
 			$ kraft events ID
 		`),
 		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "run",
+			cmdfactory.AnnotationHelpGroup:  "run",
+			cmdfactory.AnnotationHelpHidden: "true",
 		},
 	})
 	if err != nil {

--- a/internal/cli/kraft/fetch/fetch.go
+++ b/internal/cli/kraft/fetch/fetch.go
@@ -75,7 +75,8 @@ func NewCmd() *cobra.Command {
 			# Fetch a project at a given path
 			$ kraft fetch path/to/app`),
 		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "build",
+			cmdfactory.AnnotationHelpGroup:  "build",
+			cmdfactory.AnnotationHelpHidden: "true",
 		},
 	})
 	if err != nil {

--- a/internal/cli/kraft/kraft.go
+++ b/internal/cli/kraft/kraft.go
@@ -109,13 +109,11 @@ func NewCmd() *cobra.Command {
 	cmd.AddGroup(&cobra.Group{ID: "kraftcloud-svc", Title: "KRAFT CLOUD SERVICE GROUP COMMANDS"})
 	cmd.AddGroup(&cobra.Group{ID: "kraftcloud-scale", Title: "KRAFT CLOUD AUTOSCALE COMMANDS"})
 
-	cmd.AddGroup(&cobra.Group{ID: "misc", Title: "MISCELLANEOUS COMMANDS"})
-	cmd.AddCommand(login.NewCmd())
-	cmd.AddCommand(version.NewCmd())
-
 	cmd.AddGroup(&cobra.Group{ID: "compose", Title: "COMPOSE COMMANDS"})
 	cmd.AddCommand(compose.NewCmd())
 
+	cmd.AddCommand(login.NewCmd())
+	cmd.AddCommand(version.NewCmd())
 	cmd.AddCommand(x.NewCmd())
 
 	return cmd

--- a/internal/cli/kraft/login/login.go
+++ b/internal/cli/kraft/login/login.go
@@ -38,9 +38,6 @@ func NewCmd() *cobra.Command {
 			# Login to a remote service
 			$ kraft login https://github.com
 		`),
-		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "misc",
-		},
 	})
 	if err != nil {
 		panic(err)

--- a/internal/cli/kraft/set/set.go
+++ b/internal/cli/kraft/set/set.go
@@ -77,7 +77,8 @@ func NewCmd() *cobra.Command {
 			$ kraft set -w path/to/app LIBDEVFS_DEV_STDOUT=/dev/null LWIP_TCP_SND_BUF=4096
 		`),
 		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "build",
+			cmdfactory.AnnotationHelpGroup:  "build",
+			cmdfactory.AnnotationHelpHidden: "true",
 		},
 	})
 	if err != nil {

--- a/internal/cli/kraft/unset/unset.go
+++ b/internal/cli/kraft/unset/unset.go
@@ -75,7 +75,8 @@ func NewCmd() *cobra.Command {
 			$ kraft unset -w path/to/app LIBDEVFS_DEV_STDOUT LWIP_TCP_SND_BUF
 		`),
 		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "build",
+			cmdfactory.AnnotationHelpGroup:  "build",
+			cmdfactory.AnnotationHelpHidden: "true",
 		},
 	})
 	if err != nil {

--- a/internal/cli/kraft/version/version.go
+++ b/internal/cli/kraft/version/version.go
@@ -29,9 +29,6 @@ func NewCmd() *cobra.Command {
 			# Show kraft version information
 			$ kraft version
 		`),
-		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "misc",
-		},
 	})
 	if err != nil {
 		panic(err)

--- a/internal/cli/kraft/x/exp.go
+++ b/internal/cli/kraft/x/exp.go
@@ -23,7 +23,8 @@ func NewCmd() *cobra.Command {
 		Use:    "x [SUBCOMMAND]",
 		Hidden: true,
 		Annotations: map[string]string{
-			cmdfactory.AnnotationHelpGroup: "experimental",
+			cmdfactory.AnnotationHelpGroup:  "experimental",
+			cmdfactory.AnnotationHelpHidden: "true",
 		},
 	})
 	if err != nil {

